agentops/client.py

```diff
@@ -32,20 +31,22 @@ class Client(metaclass=MetaClient):
     Client for AgentOps service.
 
     Args:
-        api_key (str, optional): API Key for AgentOps services. If none is provided, key will 
-            be read from the AGENTOPS_API_KEY environment variable.
+        api_key (str, optional): API Key for AgentOps services.
+        parent_key (str, optional): Organization key to give visibility of all user sessions the user's organization.
         tags (List[str], optional): Tags for the sessions that can be used for grouping or 
             sorting later (e.g. ["GPT-4"]).
-        endpoint (str, optional): The endpoint for the AgentOps service. Defaults to 'https://api.agentops.ai'.
+        endpoint (str, optional): The endpoint for the AgentOps service.
         max_wait_time (int, optional): The maximum time to wait in milliseconds before flushing the queue. 
             Defaults to 1000.
         max_queue_size (int, optional): The maximum size of the event queue. Defaults to 100.
         override (bool): Whether to override and LLM calls to emit as events.
+        auto_start_session (bool): Whether to start a session automatically when the client is created.
```

Comment:
We should mention all default values.
I know this is unrelated to your change, but can you change max_wait_time's say "Defaults to 30,000 (30 seconds)"